bolt/bolt-protocol-message-specification-4.md

```diff
@@ -65,7 +65,7 @@ See, [**Bolt Protocol Server State Specification Version 4**]({% link bolt/bolt-
 
 ### Server Signals
 
-Jump ahead is that the signal will imediatly be available before any messages are processed in the message queue.
+Jump ahead is that the signal will immediately be available before any messages are processed in the message queue.
```

Comment:
Jump ahead means that the signal will immediately be available before any other messages...